<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherStation</class>
 <widget class="QMainWindow" name="WeatherStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather Station for RPi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temp_lcd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tinos Nerd Font</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>18.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLabel" name="deg_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Mono</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="humidity_lcd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tinos Nerd Font</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>100.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pressure_lcd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tinos Nerd Font</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>1013.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>1013</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lux_lcd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tinos Nerd Font</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>52.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>52</number>
    </property>
   </widget>
   <widget class="QLabel" name="humidity_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Mono</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="pressure_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Mono</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>hPa</string>
    </property>
   </widget>
   <widget class="QLabel" name="lux_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Mono</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LUX</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
